src/lara-plugin/plugins/embeddable-runtime-context.ts

```diff
@@ -22,7 +22,7 @@ const getReportingUrl = (
   return interactiveStatePromise.then(interactiveState => {
     try {
       const rawJSON = JSON.parse(interactiveState.raw_data);
-      if (rawJSON && rawJSON.lara_options && rawJSON.lara_options.reporting_url) {
+      if (rawJSON?.lara_options?.reporting_url) {
```

Comment:
Note on this module:
Scott requested that I mark all code additions inside of the `lara-plugin` folder with the following:
`// ACTIVITY_PLAYER_CODE:`
And to comment out instead of removing any existing LARA code and mark the commented code with the following:
`// LARA_CODE`
I believe this is so we can more easily compare the activity player code with the original code in the LARA project and potentially unify the two code bases.